packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebViewClient.java

```diff
@@ -157,6 +158,40 @@ private void notifyOnNavigationRequest(
     }
   }
 
+  void onReceivedHttpAuthRequest(final HttpAuthHandler handler, String host, String realm) {
+    HashMap<String, String> arguments = new HashMap<>();
+    arguments.put("host", host);
+    arguments.put("realm", realm);
+    methodChannel.invokeMethod(
+        "onReceivedHttpAuthRequest",
+        arguments,
+        new MethodChannel.Result() {
+          @Override
+          public void success(Object o) {
+            if (o instanceof Map) {
+              Map<?, ?> map = (Map<?, ?>) o;
+              Object username = map.get("username");
+              Object password = map.get("password");
+              if (username != null && password != null) {
```

Comment:
I don't know. How would I report that? This is basically handling the response from dart where it should definitely always be the case that `username` and `password` were sent. I am just checking it because the types are always optional when you send them over the MethodChannel.

Reply:
If the Dart side cannot send null, you can just assert that they aren't null rather than checking.